_data/retail.yml

```diff
@@ -242,3 +242,10 @@ websites:
       twitter: Zappos
       img: zappos.png
       tfa: No
+
+    - name: Zazzle
+      url: http://www.zazzle.com
+      img: zazzle.png
+      tfa: No
+      twitter: zazzle
+      facebook: zazzle
```

Comment:
If using notepad++ you can use [this plugin](https://github.com/editorconfig/editorconfig-notepad-plus-plus#install-from-the-plugin-manager), for Sublime you can use [this](https://github.com/sindresorhus/editorconfig-sublime#install), Atom has it built in by default.